keylime/src/crypto.rs

```diff
@@ -1466,4 +1512,50 @@ mod tests {
         let r = generate_tls_context(&loaded_a, &privkey, loaded_list);
         assert!(r.is_ok());
     }
+
+    #[test]
+    fn test_x509_rsa() {
+        for length in [2048, 3072, 4096] {
```

Comment:
I guess this would increase CI running time rather significantly; do we really want to check 3072+ bits key generation? If so maybe you might want to add [`#[ignore]`](https://doc.rust-lang.org/book/ch11-02-running-tests.html#ignoring-some-tests-unless-specifically-requested).

Reply:
I think I'll create a duplicate for the long keys and mark it with `#[ignore]`.